igvm/vm.py

```diff
@@ -849,3 +866,94 @@ def vm_performance_value(self) -> float:
         ) * hv_perffactor_src
 
         return float(vm_performance_value)
+
+    def aws_get_instances_overview(self, timeout: int = 5) -> List:
+        """AWS Get Instances Overview
+
+        Load or download the latest instances.json, which contains
+        a complete overview about all instance_types, their configuration,
+        performance and pricing.
+
+        :param: timeout: Timeout value for the head/get request
+
+        :return: VM types overview as list
+        """
+
+        url = AWS_INSTANCES_OVERVIEW_URL
+        file = Path(AWS_INSTANCES_OVERVIEW_FILE)
+        etag_file = Path(AWS_INSTANCES_OVERVIEW_FILE_etag)
+
+        head_req = Request(url, method='HEAD')
+        resp = urlopen(head_req, timeout=timeout)
```

Comment:
if you want this graceful i would wrap it into a proper try..except block. otherwise it can still abort the vm build

Reply:
I wrapped it and hopefully caught all possible Exceptions here.